src/Microsoft.VisualStudio.Editor.Razor/RazorCompletionItem.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal sealed class RazorCompletionItem
```

Comment:
Had to build our own completion item abstraction for the following reasons:
- Roslyn's abstraction is used in their layering over platform abstractions but is discouraged from being used (new completion APIs are the recommended ones now)
- New completion APIs (`RazorDirectiveCompletionSource`) have their own completion item concept
- A Razor language server would have its own abstraction to follow protocol.